tests/tenancy_test.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 func TestTenancyTwoOperatorsDifferentNamespaces(t *testing.T) {
+	t.Skip("Test is temporarily disabled due to KUDO v0.10.0 incompatibility")
```

Comment:
what if instead of skipping the whole test, we install the second instance with `installCrds=false` and `createRBAC=false` so that we don't install cluster-scoped resources?

Reply:
I tried this approach and it worked even without disabling RBAC (which is strange, since `ClusterRole` and `ClusterRoleBinding` are also cluster-scoped resources), but tests are still failing: `TestCRDsDeletion` fails because now CRDs are deleted together with the first operator, even though the second operator is still running.